impl/http/src/main/java/io/serverlessworkflow/impl/executors/http/AbstractRequestSupplier.java

```diff
@@ -36,20 +39,39 @@ public AbstractRequestSupplier(boolean redirect) {
   public WorkflowModel apply(
       Builder request, WorkflowContext workflow, TaskContext task, WorkflowModel model) {
     HttpModelConverter converter = HttpConverterResolver.converter(workflow, task);
+
+    if (!redirect) {
+      // disable automatic redirects handling from Jersey client
+      request.property("jersey.config.client.followRedirects", false);
```

Comment:
We can't tie our implementation to this client. Dowstream client code can override the HTTP client factory, hence it won't be Jersey.

We could have a supplier here where the downstream client can be called to react to this property instead.

By default, we set the jersey property or let the supplier from the underlying client do so.